pierone/cli.py

```diff
@@ -264,41 +263,44 @@ def describe(config, team, artifact, tag, url):
 
-    line_size, _ = shutil.get_terminal_size(100)
-    click.secho("General Information".ljust(line_size), fg='black', bg='white')
-    click.echo("Team                   ┃ {}".format(team))
-    click.echo("Artifact               ┃ {}".format(artifact))
-    click.echo("Tag                    ┃ {}".format(tag))
-    click.echo("Author                 ┃ {created_by}".format_map(tag_info))
-    click.echo("Created in             ┃ {created}".format_map(tag_info))
+    details_box = DetailsBox()
+    details_box.set("General Information", "Team", team)
+    details_box.set("General Information", "Artifact", artifact)
+    details_box.set("General Information", "Tag", tag)
+    details_box.set("General Information", "Author", tag_info["created_by"])
+    details_box.set("General Information", "Created in", tag_info["created"])
     if scm_source:
-        click.secho("Commit Information".ljust(line_size), fg='black', bg='white')
-        click.echo("Repository             ┃ {url}".format_map(scm_source))
-        click.echo("Hash                   ┃ {revision}".format_map(scm_source))
-        click.echo("Time                   ┃ {created}".format_map(scm_source))
-        click.echo("Author                 ┃ {author}".format_map(scm_source))
-        click.echo("Status                 ┃ {status}".format_map(scm_source))
-        click.secho("Compliance Information".ljust(line_size), fg='black', bg='white')
-        click.echo("Valid SCM Source       ┃ {valid}".format_map(scm_source))
+        details_box.set("Commit Information", "Repository", scm_source["url"])
+        details_box.set("Commit Information", "Hash", scm_source["revision"])
+        details_box.set("Commit Information", "Time", scm_source["created"])
+        details_box.set("Commit Information", "Author", scm_source["author"])
+        details_box.set("Commit Information", "Status", scm_source["status"])
+        details_box.set("Compliance Information", "Valid SCM Source", scm_source["valid"])
     else:
-        click.secho("Compliance Information".ljust(line_size), fg='black', bg='white')
-        click.echo("Valid SCM Source       ┃ No SCM Source")
-    click.echo("Effective Status       ┃ {}".format(effective_status))
-    click.echo("Checker Status         ┃ {}".format(checker_status))
-    click.echo("Checker Status Date    ┃ {checker_status_received_at}".format_map(tag_info))
-    click.echo("Checker Status Reason  ┃ {checker_status_reason}".format_map(tag_info))
-    click.echo("Checker Status Details ┃ {}".format(status_details.pop(0) if status_details else ""))
-    for line in status_details:
-        click.echo("                       ┃ {}".format(line))
-    if tag_info["user_status"]:
+        details_box.set("Compliance Information", "Valid SCM Source", "No SCM Source")
+    details_box.set("Compliance Information", "Effective Status", effective_status)
+    details_box.set("Compliance Information", "Checker Status", checker_status)
```

Comment:
What about  `Compliance Checker Status` ?

Reply:
Checker Status is the Compliance Checker Status